libraries/AP_Logger/LogFile.cpp

```diff
@@ -607,34 +607,57 @@ void AP_Logger::Write_Proximity(AP_Proximity &proximity)
         return;
     }
 
-    AP_Proximity::Proximity_Distance_Array dist_array {};
-    proximity.get_horizontal_distances(dist_array);
+    AP_Proximity::Proximity_Distance_Array dist_array {}, filt_dist_array {};
```

Comment:
We don't usually use `,` in variable declarations.

Similarly elsewhere

Reply:
let's remove the space between "dist_array" and the curly brackets.